net/ndproxy/src/opnsense/service/templates/OPNsense/Ndproxy/rc.conf.d/ndproxy

```diff
@@ -0,0 +1,19 @@
+# DO NOT EDIT THIS FILE -- OPNsense auto-generated file
+{% set generalSettings = helpers.getNodeByTag('OPNsense.ndproxy.general') %}
+{% if generalSettings.enabled|default("0") == "1" %}
+ndproxy_enable="YES"
+{%     if generalSettings.ndproxy_uplink_interface %}
+ndproxy_uplink_interface="{{ helpers.physical_interface(generalSettings.ndproxy_uplink_interface) }}"
```

Comment:
I think there is a slight caveat here at the moment for PPPoE (until 25.1 is out) and STF always (6RD and 6TO4) because physical interface will mismatch. compare to get_real_interface() code in core -- but eventually we need to fix that but not right now (just a comment)